packages/api/app/Utils/Algolia/core.js

```diff
@@ -2,11 +2,10 @@ const Config = use('Config');
 const AlgoliaSearch = use('App/Services/AlgoliaSearch');
 
 const config = Config.get('algolia');
-const getNestedValue = require('../getNestedValue');
 
 const initIndex = (entity) => {
-	const indexName = getNestedValue(config.indexes, entity) || entity;
-	return AlgoliaSearch.initIndex(indexName);
+	const index = config.indexes[entity];
+	return AlgoliaSearch.initIndex(index?.indexName || index);
```

Comment:
We were just repeating "indexName" the whole time. Instead, it is better to take this property **by default**.

Note: if it is an index that does not exist in the config file, such as a **replica index**, we just ignore the indexName and pass the initial value (_string_) as the parameter value